llvm-spirv/lib/SPIRV/SPIRVToLLVMDbgTran.cpp

```diff
@@ -50,7 +50,6 @@
 #include "llvm/IR/DebugProgramInstruction.h"
 #include "llvm/IR/IntrinsicInst.h"
 #include "llvm/IR/Module.h"
-#include <bit>
```

Comment:
This is approved in https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/2628 as well, so we will pulldown it later when it is merged.